modules/ui/widgets/mw.cx.ui.PageTitleWidget.js

```
@@ -90,11 +91,13 @@ mw.cx.ui.PageTitleWidget.prototype.validateTitle = function ( value ) {
 };
 
 mw.cx.ui.PageTitleWidget.prototype.getExistingTitleWarning = function () {
+	var href = mw.util.getUrl( this.getValue() );
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)